### DISCUSSION TITLE
Using Postgresql15, system page says I'm not

### DISCUSSION THREAD

**Discussion by SoleInvictus** [TandoorRecipes/recipes#2844] [Q&A]
I just started with a fresh install of Tandoor with a shiny new instance of Postgresql15. Everything appears to work (beyond a formatting issue I reported as a problem) but the System page claims I'm NOT using a Postgres back end. Just for giggles, I shut down the docker containing my Postgres database and sure enough, Tandoor doesn't work when it's down, so I'm pretty sure I actually AM using it.

Is this just a quirk? Did I do something wrong?

![Screenshot_20231229_095523_Vivaldi Browser](https://github.com/TandoorRecipes/recipes/assets/39067872/3ca45ff0-13dc-43e9-8456-59b66a75595d)

**Comment by smilerz** [comment]
duplicate
